docs/api/repl.md

```diff
@@ -0,0 +1,106 @@
+# REPL
+
+The `repl` provides a Read-Eval-Print-Loop (REPL) implementation, also known as an interactive toplevel. It is a simple, interactive computer programming environment that takes single user inputs (i.e. single expressions), evaluates them, and returns the result to the user.
+
+## Launching RELP
```

Comment:
This should be **`REPL`** and not `RELP` but other than that it's a pretty good simplification.